pallets/block-reward/src/migrations.rs

```diff
@@ -6,18 +6,15 @@ use frame_support::{storage_alias, weights::Weight};
 // A value placed in storage that represents the current version of the block-reward storage.
 // This value is used by the `on_runtime_upgrade` logic to determine whether we run storage
 // migration logic. This internal storage version is independent to branch/crate versions.
-#[derive(Encode, Decode, Clone, Copy, PartialEq, Eq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+#[derive(
+	Encode, Decode, Clone, Copy, Default, PartialEq, Eq, RuntimeDebug, TypeInfo, MaxEncodedLen,
+)]
 pub enum StorageReleases {
+	#[default]
```

Comment:
Could I ask why this include in this MR?

Reply:
Yes, could have been done in another PR, but I applied cargo clippy and tought, why not directly, because it's not a big thing...